recipes/requests_pkcs12/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "requests_pkcs12" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f9fe10356c5d7464bb5e06125f76d3aaba5914f2c8ccb48b3f466d0f1d21996f
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pyopenssl >=0.14
+    - requests >=2.18.4
+
+test:
+  imports:
+    - requests_pkcs12
+
+about:
+  home: https://github.com/m-click/requests_pkcs12
+  license: ISC
+  license_family: BSD
```

Comment:
ISC is functionally equivalent to the simplified BSD and MIT licenses according to [wikipedia](https://en.wikipedia.org/wiki/ISC_license). 

Please remove `license_family` if BSD or MIT are not good options here.